<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_main</class>
 <widget class="QMainWindow" name="window_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ENE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="widget_tab">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_local_files">
       <attribute name="title">
        <string>Local Files</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QStackedWidget" name="stack_local_files">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>Streams</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_browser">
       <attribute name="title">
        <string>Browse</string>
       </attribute>
       <widget class="QComboBox" name="combobox_season">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>167</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentText">
         <string>All</string>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Winter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spring</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Summer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fall</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_season">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Season</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_filter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Filters</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combobox_sort">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentText">
         <string>Sort</string>
        </property>
        <item>
         <property name="text">
          <string>Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Popularity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Score</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trending</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Favourites</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date Added</string>
         </property>
        </item>
       </widget>
       <widget class="QToolButton" name="button_sort_order">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>230</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="combobox_format">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>167</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentText">
         <string>Format</string>
        </property>
        <item>
         <property name="text">
          <string>Format</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TV Short</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Movie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Special</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OVA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ONA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="combobox_status">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>290</y>
          <width>167</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentText">
         <string>Status</string>
        </property>
        <item>
         <property name="text">
          <string>Status</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Releasing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Finished</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Yet Released</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cancelled</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="combobox_streaming">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>320</y>
          <width>167</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentText">
         <string>Streaming On</string>
        </property>
        <item>
         <property name="text">
          <string>Streaming On</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crunchyroll</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Funimation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Netflix</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Amazon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hidive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hulu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Animelab</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="combobox_genre_tag">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>350</y>
          <width>167</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentText">
         <string>Genres &amp; Tags</string>
        </property>
        <item>
         <property name="text">
          <string>Genres &amp; Tags</string>
         </property>
        </item>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>1211</width>
          <height>781</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridlayout_season"/>
       </widget>
       <widget class="QGroupBox" name="groupbox_year">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>191</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Year</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinbox_year_min">
           <property name="minimum">
            <number>1950</number>
           </property>
           <property name="maximum">
            <number>2020</number>
           </property>
           <property name="value">
            <number>2018</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinbox_year_max">
           <property name="minimum">
            <number>1950</number>
           </property>
           <property name="maximum">
            <number>2020</number>
           </property>
           <property name="value">
            <number>2018</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>My List</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="action_open_folder"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
    <addaction name="action_prefences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Ene"/>
    <addaction name="action_source_code"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_prefences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action_open_folder">
   <property name="text">
    <string>&amp;Open local folder</string>
   </property>
  </action>
  <action name="actionAbout_Ene">
   <property name="text">
    <string>&amp;About Ene</string>
   </property>
  </action>
  <action name="action_source_code">
   <property name="text">
    <string>&amp;Source Code</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
